Sources/DotLottie/Public/dotlottie_player.swift

```diff
@@ -731,6 +733,16 @@ public class DotLottiePlayer:
         )
     }
 
+    public func seek(no: Float) -> Bool {
+        return try! FfiConverterBool.lift(
```

Comment:
What is the possibility that this force try! would fail and crash?

maybe can use try? and add `|| false` at the end to avoid the possible crash?